### PR TITLE
Add lock file options for nuget.exe

### PR DESCRIPTION
**Pull request by nkolev92**
Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/1791

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[d502dc6](https://github.com/NuGet/docs.microsoft.com-nuget/commits/d502dc619a80a535bd6b2302e2ac5444d7b8332b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-addLockFileOptions/docs/consume-packages/Package-References-in-Project-Files.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files?branch=pr-en-us-1921) |
[docs/reference/cli-reference/cli-ref-restore.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-addLockFileOptions/docs/reference/cli-reference/cli-ref-restore.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-restore?branch=pr-en-us-1921) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C28%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002281950037149-1921%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=HTuDnLw95CH26HplbfcrpWiCC%2Bb5rayv9ZjqDsa1Ax4%3D&st=2020-02-28T19%3A45%3A43Z&se=2020-03-30T19%3A50%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/consume-packages/Package-References-in-Project-Files.md [hunk 1]
```diff
@@ -247,9 +249,9 @@ If `ProjectA` has a dependency on a `PackageX` version `2.0.0` and also referenc
 
 You can control various behaviors of restore with lock file as described below:
 
-| Option | MSBuild equivalent option | Description|
-|:---  |:--- |:--- |
-| `--use-lock-file` | RestorePackagesWithLockFile | Opts into the usage of a lock file. | 
-| `--locked-mode` | RestoreLockedMode | Enables locked mode for restore. This is useful in CI/CD scenarios where you want repeatable builds.|   
-| `--force-evaluate` | RestoreForceEvaluate | This option is useful with packages with floating version defined in the project. By default, NuGet restore will not update the package version automatically upon each restore unless you run restore with this option. |
-| `--lock-file-path` | NuGetLockFilePath | Defines a custom lock file location for a project. By default, NuGet supports `packages.lock.json` at the root directory. If you have multiple projects in the same directory, NuGet supports project specific lock file `packages.<project_name>.lock.json` |
+| NuGet.exe option | dotnet pption | MSBuild equivalent option | Description |
```
Comment (zkat) [inline docs/consume-packages/Package-References-in-Project-Files.md]:
```suggestion
| NuGet.exe option | dotnet option | MSBuild equivalent option | Description |
```

File: docs/consume-packages/Package-References-in-Project-Files.md [hunk 2]
```diff
@@ -247,9 +249,9 @@ If `ProjectA` has a dependency on a `PackageX` version `2.0.0` and also referenc
...
+|:--- |:--- |:--- |:--- |
+| `-UseLockFile` |`--use-lock-file` | RestorePackagesWithLockFile | Opts into the usage of a lock file. |
+| `-LockedMode` | `--locked-mode` | RestoreLockedMode | Enables locked mode for restore. This is useful in CI/CD scenarios where you want repeatable builds.|   
+| `-ForceEvaluate` | `--force-evaluate` | RestoreForceEvaluate | This option is useful with packages with floating version defined in the project. By default, NuGet restore will not update the package version automatically upon each restore unless you run restore with this option. |
+| `-LockFilePath` | `--lock-file-path` | NuGetLockFilePath | Defines a custom lock file location for a project. By default, NuGet supports `packages.lock.json` at the root directory. If you have multiple projects in the same directory, NuGet supports project specific lock file `packages.<project_name>.lock.json` |
```
Comment (zivkan) [inline docs/consume-packages/Package-References-in-Project-Files.md]: Would it be better if this table was sorted?

Comment (nkolev92) [reply docs/consume-packages/Package-References-in-Project-Files.md]: I think we should close in favor of https://github.com/NuGet/docs.microsoft.com-nuget/pull/1921...I know it's later, but I am actively working on it, so it stands a better chance at working.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[5a51d8c](https://github.com/NuGet/docs.microsoft.com-nuget/commits/5a51d8c02379b8326a4850114e8db59ba29ec11d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Package-References-in-Project-Files.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-addLockFileOptions/docs/consume-packages/Package-References-in-Project-Files.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files?branch=pr-en-us-1921) |
[docs/reference/cli-reference/cli-ref-restore.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master-nkolev92-addLockFileOptions/docs/reference/cli-reference/cli-ref-restore.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-restore?branch=pr-en-us-1921) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C28%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002282005459265-1921%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=wBp3PMS35tJm2YwO6mBN17g4AvHzXgpZC9reqhVSxJ0%3D&st=2020-02-28T20%3A01%3A44Z&se=2020-03-30T20%3A06%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/consume-packages/Package-References-in-Project-Files.md (hunk 2 above)
Comment (nkolev92) [reply docs/consume-packages/Package-References-in-Project-Files.md]: I thought Iw as replying on a different issue :D
I think they are ordered in importance order, I think that's ok.

**Pull request merged**